hw-L11-solid/src/main/java/com/abrenchev/AtmImpl.java

```diff
@@ -0,0 +1,68 @@
+package com.abrenchev;
+
+import com.abrenchev.exceptions.AtmWithdrawException;
+import com.abrenchev.interfaces.Atm;
+
+import java.util.*;
+
+public class AtmImpl implements Atm {
+    private Map<BanknoteValue, AtmCell> banknoteStorage = new HashMap<>();
```

Comment:
Можно, в принципе, обойтись List<AtmCell>
```suggestion
    private List<AtmCell> banknoteStorage = new ArrayList<>();
```